Aptoide/aptoide-client-v8
# [ASV-2220] Read more detail view

**What does this PR do?**

   This PR aims at fixing the dark theme in the read more description detail view inside AppView. Also took the chance to fix the toolbar inside this fragment as it was following a deprecated business rule. Before, we were setting the toolbar in read more with the color of the store where that same app was. Now we set the normal color (same as appview). This was discussed with Luis Guilherme.

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] DescriptionFragment.java

**How should this be manually tested?**

  Open the read more of several apps, and check that it has the normal color and never the color of the store where that app is.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2220](https://aptoide.atlassian.net/browse/ASV-2220)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
